src/reactive-membrane.ts

```diff
@@ -76,31 +90,150 @@ const defaultValueMutated: ReactiveMembraneMutationCallback = (obj: any, key: Pr
 };
 const defaultValueDistortion: ReactiveMembraneDistortionCallback = (value: any) => value;
 
-export function wrapDescriptor(membrane: ReactiveMembrane, descriptor: PropertyDescriptor, getValue: (membrane: ReactiveMembrane, originalValue: any) => any): PropertyDescriptor {
-    const { set, get } = descriptor;
+const reserveGetterMap = new WeakMap<() => any, () => any>();
```

Comment:
the reverse maps are used to unwrap accessor descriptors back into their original values, we were leaking membrane generated accessor wrappers into the original target when redefining properties :(